acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/privacy/multitenancy/MultiTenancyAcceptanceTest.java

```diff
@@ -233,6 +234,7 @@ public void privGetTransactionReceiptSuccessShouldReturnTransactionReceiptAfterM
     node.verify(priv.getSuccessfulTransactionReceipt(transactionReceipt));
   }
 
+  @Ignore // TODO: how has that ever worked?
```

Comment:
Yup, I was wondering how that test has ever worked. I will have another look ...

Reply:
I have fixed the test. The tests in this class are not very nice. I have fixed the one that I did ignore, but I think that some of the other tests could be quite brittle as well. 
In this case the stubbing was returning a privacy group for a Pantheon group, not eea, and some other things were not quite right. 
The test could only have worked because the stubbing was consistently doing the wrong thing, like it was sending a Pantheon private transaction etc.